modules/saml/www/sp/saml1-acs.php

```diff
@@ -66,7 +68,7 @@
     $responseXML = base64_decode($responseXML);
     $isValidated = false; /* Must check signature on response. */
 } else {
-    assert(false);
```

Comment:
This leads to undefined variables below.. If we can't handle a response because something is missing, we should just raise an exception.